integration-tests/ios.spec-int.js

```diff
@@ -153,7 +149,7 @@ if (!process.env.CI) {
             })
         }
     })
-    test.describe('primary screen screenshots', () => {
+    test.describe.skip('primary screen screenshots', () => {
```

Comment:
❓ Are we expecting to skip this test?

Reply:
@alistairjcbrown  yeah, because the following PR for iOS will change this, and for now it was just introducing noise (and even more screenshots). Everything that was covered here is covered elsewhere luckily :)

BTW, I think we can move to chromatic screenshot testing soon for this stuff. but until we do that, this amount of screenshots is genuinely helping me make changes more quickly 🤞